lib/modules/ipfs/upload.js

```diff
@@ -44,6 +45,30 @@ class IPFS {
       function printUrls(dir_hash, callback) {
         console.log(("=== " + __("DApp available at") + " http://localhost:8080/ipfs/" + dir_hash + "/").green);
         console.log(("=== " + __("DApp available at") + " https://ipfs.infura.io/ipfs/" + dir_hash + "/").green);
+        if(self.env === 'development') {
+          console.log(("\n=== " +
+            "Blockchain must be running".bold + 
+            " ===").yellow);
+          console.log((
+            "embark run".italic + 
+            __(" or ") + 
+            "embark blockchain".italic + 
```

Comment:
Do you think we should add `simulator` here too?

Reply:
no, let's discourage the use of the ismulator